.vscode/settings.json

```diff
@@ -1,10 +1,10 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
Why the changes?

Reply:
VSCode/eslint does this by default in the newest version, so this is right.